lib/twilio/call.ts

```diff
@@ -215,6 +215,12 @@ class Call extends EventEmitter {
    */
   private _mediaStatus: Call.State = Call.State.Pending;
 
+  /**
+   * A map of messages sent via sendMessage API.
+   * The message will be deleted once an 'ack' or an error is received from the server.
+   */
+  private _messages: Map<string, Call.Message> = new Map();
```

Comment:
I'm assuming the `string` is the `voiceEventSid` of the message? Should we write that down?